_data/cloudservices.yml

```diff
@@ -3177,8 +3174,8 @@ services:
             ref: https://cloud.google.com/dataflow/
             icon: Cloud-Dataflow.png
       - ibm:
-          - name: Streaming Analytics
-            ref: https://cloud.ibm.com/docs/services/StreamingAnalytics?topic=StreamingAnalytics-gettingstarted
+          - name: Streaming Analytics (deprecated)
+            ref: https://cloud.ibm.com/catalog/services/streaming-analytics
```

Comment:
The deprecation notice link is 404 now

I am not sure whether [IBM Streams 5.5 for IBM Cloud Pak for Data](https://www.ibm.com/docs/en/streams/5.5) is the target, please validate

Reply:
This link still works for me; I'll leave as is unless there is something better to put here.